privaterelay/context_processors.py

```diff
@@ -59,16 +59,17 @@ def _get_csat_cookie_and_reason(request):
     reason_to_show_csat_survey = None
     csat_dismissal_cookie = ""
     if (profile.has_premium):
-        # There are two reasons why someone might not have a subscription date set:
-        # - They subscribed before we started tracking that.
-        # - They have Premium because they have a Mozilla email address.
-        # In the latter case, their first visit date is effectively their
-        # subscription date. In the former case, they will have had Premium for
-        # a while, so they can be shown the survey too. Their first visit will
-        # have been a while ago, so we'll just use that as a proxy for the
-        # subscription date:
-        days_since_subscription = (datetime.now(timezone.utc) - datetime.fromisoformat(first_visit)).days
-        if (profile.date_subscribed):
+        if not profile.date_subscribed:
+            # There are two reasons why someone might not have a subscription date set:
+            # - They subscribed before we started tracking that.
+            # - They have Premium because they have a Mozilla email address.
+            # In the latter case, their first visit date is effectively their
+            # subscription date. In the former case, they will have had Premium for
+            # a while, so they can be shown the survey too. Their first visit will
+            # have been a while ago, so we'll just use that as a proxy for the
+            # subscription date:
+            days_since_subscription = (datetime.now(timezone.utc) - datetime.fromisoformat(first_visit)).days
```

Comment:
It's exiting code, but if the cookie isn't set, `first_visit` is `None`, so `datetime.fromisoformat(None)` fails.